gobject-introspection.yaml

```diff
@@ -76,7 +80,7 @@ subpackages:
         - gobject-introspection
         - libtool
         - posix-libc-utils
-        - py3-setuptools
+        - py${{vars.py-version}}-setuptools
         - python3
```

Comment:
drop `python3` here. melange will add the necessary depends for it. (on python-3.13-base).